runner/runner.go

```diff
@@ -0,0 +1,16 @@
+package runner
+
+// RunnerStatus that represents the Status of a Runner
+type RunnerStatus int
+
+// RunnerStatus implementation as enum
+const (
```

Comment:
@fforootd 
and here the runner status

Reply:
i did reduce the api side to the following 

```
enum Status {
    UNKNOWN = 0;
    ACTIVE = 1;
    PAUSED = 2;
    ONLINE = 3;
    OFFLINE = 4;
}
```

Reply:
this alignment with the gitlab-runner status conditions works for me so far 👍